class/defaults.yml

```diff
@@ -242,7 +242,7 @@ parameters:
         CpuCapacity:
           rules:
             ClusterCpuUsageHigh:
-              enabled: true
+              enabled: false
```

Comment:
I don't understand why we disable this alert by default. As an operator I would want to know if my cluster has high CPU usage.

Also, this change should probably be in a different PR, this PR currently contains two unrelated changes:

1. Disable predictive alerts by default
2. Disable ClusterCpuUsageHigh by default